src/js/components/TaskTable.js

```diff
@@ -39,9 +40,55 @@ class TaskTable extends React.Component {
   }
 
   getStatusValue(task) {
+    let taskHealth = this.getTaskHealth(task);
+    if (taskHealth === true) {
```

Comment:
What is the purpose of checking specifically for the boolean `true` and `false` values? Would coercion not be sufficient here? (e.g. `if (taskHealth)` and `if (!taskHealth)`). Just curious :)